packages/relay/src/QueryRenderer.js

```diff
@@ -4,11 +4,8 @@
 import * as React from 'react';
 import { StyleSheet, View } from 'react-native';
 import { Loader } from '@kiwicom/universal-components';
-import {
-  QueryRenderer,
-  type ReadyState,
-  type GraphQLTaggedNode,
-} from 'react-relay';
+import { QueryRenderer as KiwiQueryRenderer } from '@kiwicom/relay';
+import { type ReadyState, type GraphQLTaggedNode } from 'react-relay';
```

Comment:
I think you don't need `ReadyState` - just use the props from `KiwiQueryRenderer` (`onSystemError` and `onLoading`). There is nothing special in this project preventing you from using them (I think).